ppr-ui/src/components/tables/common/TableRow.vue

```diff
@@ -300,6 +304,8 @@
             class="edit-btn"
             color="primary"
             @click="editDraft(item)"
+            aria-hidden="false"
```

Comment:
Is this required on the buttons? I thought they are aria-hidden by default?

Reply:
I will remove it.